com.unity.render-pipelines.high-definition/Editor/Lighting/LightmappingHDRP.cs

```diff
@@ -0,0 +1,156 @@
+using System;
+using JetBrains.Annotations;
+using UnityEngine;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Rendering.HighDefinition;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    /// <summary>
+    ///     Lightmapping utilities for HDRP
+    /// </summary>
+    public static class LightmappingHDRP
+    {
+        /// <summary>
+        ///     Bake the <paramref name="probe" /> and updated its baked texture.
+        /// </summary>
+        /// <param name="probe">The probe to bake.</param>
+        /// <param name="path">The asset path to write the baked texture to.</param>
+        /// <param name="options">The options to use for the bake.</param>
+        /// <returns>
+        ///     <c>null</c> on success or the error that occured.
+        ///     The error can be:
+        ///     * <see cref="ArgumentException" /> if the <paramref name="path" /> is invalid.
+        ///     * <see cref="ArgumentNullException" /> if the <paramref name="probe" /> is <c>null</c>.
+        ///     * <see cref="Exception" /> if the <paramref name="probe" /> is not supported. Only
+        ///     <see cref="HDAdditionalReflectionData" /> probes are currently supported.
```

Comment:
This functional currently only supports <see cref="HDAdditionalReflectionData" /> probes.